### PR TITLE
Docs name change for keepAlive() to maintainState

### PR DESCRIPTION
**Pull request by bdlindsay**
I assume this got renamed at some point, but I tried to use `keepAlive()` and saw that it didn't exist. I eventually found the maintainState property. Feel free to let me know of any errors or change anything yourself.

### CODE REVIEW

**Comment by Reprevise** [comment]
According to the release notes of 2.0.0-dev.2, `maintainState` was renamed to a method named `keepAlive()` which is why you no longer see `maintainState` in the docs. Though, since riverpod 2.0 isn't released to stable, there could probably be a mention of the old `maintainState` setter near the new `keepAlive` docs.

**Comment by bdlindsay** [comment]
Oh I see, I was just using the latest that pub.dev listed. I've edited my change to just include a mention of the maintainState setter

File: website/docs/concepts/modifiers/auto_dispose.mdx [hunk 1]
```diff
@@ -67,6 +67,8 @@ it, then the request will be performed again.
 But if the request completed successfully, the state will be preserved
 and re-entering the screen will not trigger a new request.
 
+Note: This is a rework of the version 1 property called `maintainState`.
```
Comment (rrousselGit) [inline website/docs/concepts/modifiers/auto_dispose.mdx:70]:
```suggestion
:::info
In version 1.0.x, the equivalent of `keepAlive` is the property called `maintainState`.
:::
```

**Comment by rrousselGit** [comment]
Thanks!

**Pull request merged**